packages/account/src/Components/poi-unverified/unverified.spec.js

```diff
@@ -0,0 +1,49 @@
+import { render, screen } from '@testing-library/react'
+import React from 'react'
+import { Unverified } from './unverified.jsx'
+import {PlatformContext} from '@deriv/shared'
+import { BrowserRouter } from 'react-router-dom'
```

Comment:
can we rearrange these imports?  Refer here (https://github.com/binary-com/deriv-app/blob/master/docs/JavaScript/README.md#import-rules)